### PR TITLE
Principle on fake consent

### LINKED ISSUE #192: Don't ask for fake consent, even if real consent is not required
Consent is difficult to obtain and should be asked for rarely.

In some situations, a party can comply with a requirement to get a person to perform some action that is treated as <q>consent</q> without obtaining the person's actual [=consent=].

For example, in some jurisdictions, the law may require "consent' but it would be possible for a party to comply with the law by obtaining "fake consent" (for example, requiring a person to click <q>Yes, I consent</q> in a situation where the person has not been fully informed of what is being consented to.)

What should parties do in a case where these principles do not require getting real consent, but the law requires some action, and will settle for fake consent?

For example, <q>There are some laws that require websites to ask their users' consent to store information even when that information doesn't affect those people's privacy. These laws are harmful.</q> (@jyasskin )

Because requests for consent should be important enough to justify attention from a [=person=], a
[=party=] should avoid distracting the user with attempts to get fake <q>consent</q> in situations
when actual [=consent=] would not be required by these principles.  Responding to unnecessary requests
for consent, real or fake, is unacceptable [=privacy labour=] and limits how much time and attention
a [=person=] can devote to meaningful [=consent=] decisions.

### PR DESCRIPTION
**Pull request by dmarti**
Simplify the fake consent problem, new language

Fixes #192

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/290.html" title="Last updated on Jul 5, 2023, 2:43 PM UTC (3551c2d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/290/86d1c1f...3551c2d.html" title="Last updated on Jul 5, 2023, 2:43 PM UTC (3551c2d)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1962,6 +1962,20 @@
...
+<div class="practice">
+<p>
+  <span class="practicelab" id="principle-fake-consent">
+    An [=actor=] should not prompt a [=person=] for consent if the
+	person is unlikely to have sufficient information to make an informed decision to consent or not.
+  </span>
+</p>
+</div>
+
+In considering whether or not a person is sufficiently informed to be promoted for consent,
```
Comment (darobin) [inline index.html:1974]: I'm not sure I get what it mean to be promoted for consent?

File: index.html [hunk 2]
```diff
@@ -1962,6 +1962,20 @@
...
+actors should be realistic in comparing [=privacy labour=] required with actual user time spent in
+the relevant context.
+Simply providing a link to a complex policy is unlikely to mean that the person is
+informed, if the policy takes half an hour to read and the average site visit is ten minutes.
```
Comment (darobin) [inline index.html:1978]: I'm not sure that I love this measure. I don't want to spend 30min reading a complex policy even for a site that I use every day!

**Comment by dmarti** [comment]
Remove number of minutes, change to reasonable

Make this into explanatory text for first principle in section

**Comment by dmarti** [comment]
@darobin Made changes that should address your comments here.

**Review by jyasskin — APPROVED** [review]

**Review by npdoty — APPROVED** [review]
looks good. works better as explanatory text, thanks for making the change.

**Review by torgo — APPROVED** [review]
lgtm

**Pull request merged**